simple-stack-example-kotlin-dagger/src/main/java/com/zhuinden/simplestackkotlindaggerexample/userdetail/UserDetailFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.zhuinden.simplestackkotlindaggerexample.userdetail
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.zhuinden.simplestackkotlindaggerexample.BaseFragment
+import com.zhuinden.simplestackkotlindaggerexample.R
+import com.zhuinden.simplestackkotlindaggerexample.realmobjects.UserRO
+import com.zhuinden.simplestackkotlindaggerexample.requireArguments
+import kotlinx.android.synthetic.main.fragment_user_detail.userEmail
+import kotlinx.android.synthetic.main.fragment_user_detail.userName
+import kotlinx.android.synthetic.main.fragment_user_detail.userPhoneNumber
+
+
+class UserDetailFragment : BaseFragment() {
+
+    private lateinit var userRO: UserRO
+
+    companion object {
+        private const val ARG_USER_RO = "userRO"
+        @JvmStatic
+        fun newInstance(userRO: UserRO) =
+            UserDetailFragment().apply {
+                arguments = Bundle().apply {
+                    putParcelable(ARG_USER_RO, userRO)
+                }
+            }
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        requireArguments.let {
+            userRO = it.getParcelable(ARG_USER_RO)
```

Comment:
You can use `getKey<UserDetailKey>().userRO` here, that way you can remove the arguments call here completely (and the static constant)